### ISSUE TITLE
gpaste-ui segfaults when save-history is disabled and ~/.local/share/GPaste is missing

### ISSUE THREAD

**Issue by vgaetera** [Keruspe/GPaste#341]
Description of problem:
gpaste-ui segfaults when `save-history` is disabled and `~/.local/share/GPaste` is missing

Package version:
```
gpaste-ui-3.40.1-1.fc34.x86_64
```

How reproducible:
Always

Steps to reproduce:
```
gsettings set org.gnome.GPaste save-history false
gio trash ~/.local/share/GPaste
```

Actual results:
```
> /usr/libexec/gpaste/gpaste-ui
Segmentation fault (core dumped)
```

**Comment by Sebastian-Hirsch** [comment]
I can confirm that this issue still exists version 3.42.1 of gpaste.

**Comment by vgaetera** [comment]
Appears to be fixed in `gpaste-ui-43.0-1.fc37.x86_64`.
